### ISSUE TITLE
NonadiabaticModels, Double Well

### ISSUE THREAD

**Issue by smjanke** [NQCD/NonadiabaticMolecularDynamics.jl#111]
https://github.com/NQCD/NonadiabaticMolecularDynamics.jl/blob/8dcd94f85b7dc6ee5e3530c90fb082cc4de258fc/docs/src/nonadiabaticmodels/overview.md#L35

Are nstates and the degrees of freedom hard-coded into the model? If not, I would specify them as arguments in the example, so it's clearer how to change them. If they are, it should be mentioned that they are hard-coded.

**Comment by jamesgardner1421** [comment]
Yes I realised that most models have fixed dofs and nstates so it makes sense to hard-code them in their definitions, rather than requiring the user to specify them. Some models can have a variable number and these can be given as arguments.

Are you sure it needs to be said that they are hard-coded? Maybe instead it would be best to describe what that the DoubleWell is (a two state 1D model that can be used to represent electron transfer in a molecule).
Then it is obvious that the number of states and ndofs is just an attribute of this particular model?
